src/audio/buffer.c

```diff
@@ -40,9 +41,12 @@ struct comp_buffer *buffer_alloc(uint32_t size, uint32_t caps, uint32_t align)
 		return NULL;
 	}
 
-	/* allocate new buffer */
+	/*
+	 * allocate new buffer, align the allocation size to a cache line for
+	 * the coherent API
+	 */
 	buffer = rzalloc(SOF_MEM_ZONE_RUNTIME_SHARED, 0, SOF_MEM_CAPS_RAM,
-			 sizeof(*buffer));
+			 ALIGN_UP(sizeof(*buffer), PLATFORM_DCACHE_ALIGN));
```

Comment:
Do we have a ```rzalloc_aligned()``` if not, would be cleaner (and it could align at beginning and end).

Reply:
@lgirdwood no, we don't, and the beginning is always aligned